### PR TITLE
Submission: Wider MLP 3x + int6 quant + sliding window eval, val_bpb=1.1659

### PR DESCRIPTION
**Pull request by jfprincz**
## Submission: Wider MLP 3x + int6 Quantization + Sliding Window Eval

**val_bpb: 1.1659** | **Total size: 14,855,508 bytes** (under 16MB)

Three orthogonal improvements over the naive baseline:

1. **Wider MLP (MLP_MULT=3.0)** - 2x to 3x expansion (hidden=1536), ~0.019 BPB improvement
2. **int6 per-row on MLP+attention** - saves ~4MB artifact space, only +0.010 BPB degradation; zstd-22 compression
3. **Sliding window eval (stride=256)** - overlapping windows, batched forward_logits, ~0.033 BPB improvement, zero artifact cost

### Run command

```bash
RUN_ID=official_v1_reach MAX_WALLCLOCK_SECONDS=600 VAL_LOSS_EVERY=0 TRAIN_LOG_EVERY=200 MATRIX_LR=0.020 SCALAR_LR=0.020 TIED_EMBED_LR=0.030 MUON_MOMENTUM=0.99 MUON_MOMENTUM_WARMUP_STEPS=1500 MUON_MOMENTUM_WARMUP_START=0.92 WARMDOWN_ITERS=3000 torchrun --standalone --nproc_per_node=8 train_gpt.py
```

### Key metrics

| Metric | Value |
|--------|-------|
| Steps (10 min cap) | 12,485 |
| int6 sliding val_bpb | **1.1659** |
| Artifact size | 14,855,508 bytes |
| Two seeds: 1.16658, 1.16591 (submitted: 1338) |

See README.md in the submission folder for full details.

### CODE REVIEW

**Comment by MatoTeziTanka** [comment]
### Community Review — Submission: Wider MLP 3x + int6 quant + sliding window eval, val_bpb=1.1659

**BPB:** 1.1659 | **Compliance:** LOOKS CLEAN — pure-neural submission, no TTT/SLOT/n-gram-cache

**What I found in the code** (head SHA `2790ac088cd3`, file `records/track_10min_16mb/2026-03-19_WiderMLP_Int6_SlidingWindow_1.1666/train_gpt.py`):

Static code review found no TTT adaptation function, no SLOT optimization loop, no n-gram-cache class, and no pre-quant val-token fine-tune. The eval path uses the standard sliding-window stride-64 pattern. The submission is a pure-neural architecture iteration on the standard SP1024/SP4096/SP8192 baseline.

**CPU smoke test (CT2038 proteus-engine, 2026-04-11):** import OK in 0.03s, dim=512, layers=9, vocab=1024, code=56014 B, SMOKE_TEST_PASS

**Verdict:** LOOKS CLEAN.

**Recommendation to @cocohearts @valerio-oai @0hq @yuzhougu-oai @notapplica:** **MERGE** pending the usual record-track checks (3-seed validation, under-16MB artifact cap, ≤600s train + ≤600s eval on 8×H100 SXM). No compliance flags from the classification pass — this looks like a clean pure-neural iteration on the standard baseline.

*Auto-classification caveat:* this review was drafted by the AST-based classifier. If there's a non-standard eval mechanism (logit postprocessing, hedge mixing, etc.) that I missed because it's factored into a helper file or a non-standard function name, please flag it and I'll re-run the audit manually.

---
*Reviewed by [@MatoTeziTanka](https://github.com/MatoTeziTanka) — [The Agora](https://matotezitanka.github.io/parameter-golf). CPU smoke test (CT2038 proteus-engine, 2026-04-11): import OK in 0.03s, dim=512, layers=9, vocab=1024, code=56014 B, SMOKE_TEST_PASS. Classification via deterministic AST-based `classify_prs.py` (pattern bank derived from ~65 manually-reviewed PRs earlier in the 2026-04-11 sweep). This review was auto-drafted from a template and spot-checked before posting — if the template misread your code, please call it out so I can iterate the classifier.*
